src/Symfony/Component/ValueExporter/ValueExporter.php

```diff
@@ -0,0 +1,93 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\ValueExporter;
+
+use Symfony\Component\ValueExporter\Exporter\ValueExporterInterface;
+use Symfony\Component\ValueExporter\Exporter\ValueToStringExporter;
+use Symfony\Component\ValueExporter\Formatter\CallableToStringFormatter;
+use Symfony\Component\ValueExporter\Formatter\DateTimeToStringFormatter;
+use Symfony\Component\ValueExporter\Formatter\FormatterInterface;
+use Symfony\Component\ValueExporter\Formatter\PhpIncompleteClassToStringFormatter;
+
+// Load the global to_string() function
+require_once __DIR__.'/Resources/functions/to_string.php';
```

Comment:
composer has the file autoload method, but I just remembered it might give issues as you'd have to register it in `symfony/symfony` as well. Therefore it might not be the best solution to remove it here (if even possible)